docs/get-started/tips-and-tricks.md

```diff
@@ -184,7 +184,7 @@ const mapRef = useRef<MapRef>();
 
 const checkIfPositionInViewport = (lat, lng) => {
     const bounds = mapRef.current.getMap().getBounds();
-    return (lat >= bounds._sw.lat && lat <= bounds._nw.lat && lng >= bounds._sw.lng && lng <= bounds._nw.lng);
+    return (lat >= bounds._sw.lat && lat <= bounds._ne.lat && lng >= bounds._sw.lng && lng <= bounds._ne.lng);
```

Comment:
Just call `bounds.contains([lng, lat])`?

https://docs.mapbox.com/mapbox-gl-js/api/geography/#lnglatbounds#contains

Reply:
That is a nice hint. I just ran into this error while pasting this code and wanted to prevent others from doing the same.